src/main/kotlin/frc/robot/subsystems/superstructure/Intake.kt

```diff
@@ -1,5 +1,64 @@
 package frc.robot.subsystems.superstructure
 
 import org.ghrobotics.lib.commands.FalconSubsystem
+import org.ghrobotics.lib.mathematics.units.nativeunit.DefaultNativeUnitModel
+import org.ghrobotics.lib.motors.ctre.FalconSRX
+import org.ghrobotics.lib.wrappers.FalconDoubleSolenoid
+import org.ghrobotics.lib.wrappers.FalconSolenoid
+
+
+var cargo = false
```

Comment:
Why is this boi just floating outside the class? While this technically works, move it into the Intake class.

Reply:
done

Reply:
If you think you're done, click "resolve conversation"